modules/site-health/admin-site-health.php

```diff
@@ -442,6 +442,17 @@ public function info( $debug_info ) {
 			}
 		}
 
+		// Multisite
+		if ( is_multisite() ) {
+			if ( is_plugin_active_for_network( POLYLANG_BASENAME ) ) {
+				$network_activated = __( 'Yes', 'polylang' );
+			} else {
+				$network_activated = __( 'No', 'polylang' );
+			}
+			$fields['network_activated']['label'] = __( 'Network Activated', 'polylang' );
+			$fields['network_activated']['value'] = $network_activated;
```

Comment:
```suggestion
			$fields['network_activated'] = array(
				'label' => __( 'Network Activated', 'polylang' ),
				'value' => $network_activated,
			);
```
Proposal with less assignments.